plugins/w3c_logs.yaml

```diff
@@ -0,0 +1,109 @@
+version: 0.0.1
+title: W3C
+description: File Input W3C Parser
+parameters:
+  - name: file_log_path
+    type: "[]string"
+    required: true
+    default: 
+      - "/var/log/zookeeper/zookeeper.log"
+  - name: exclude_file_log_path
+    type: "[]string"
+    default: 
+      - ""
+  - name: encoding
+    type: string
+    supported:
+      - utf-8
+      - utf-16le
+      - utf-16be
+      - ascii
+      - big5
+    default: utf-8
+  - name: log_type
+    type: string
+    default: w3c
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+  - name: max_concurrent_files
+    type: int
+    default: 512
+  - name: include_file_name
+    type: bool
+    default: true
+  - name: include_file_path
+    type: bool
+    default: false
+  - name: include_file_name_resolved
+    type: bool
+    default: false
+  - name: include_file_path_resolved
+    type: bool
+    default: false
+  - name: header
+    type: "string"
+    required: true 
+  - name: delimiter
+    type: string
+    default: "\t"
+  - name: header_delimiter
+    type: string
+    default: "\t"
```

Comment:
Looks like this isn't used (otel doesn't expose this option); We should remove it.